tutorial/setup/external.md

```diff
@@ -11,5 +11,7 @@ At this point in time I do not have the resources to publicly host a PlanarAlly
 Important to know is that these services are **not** operated by me and thus I cannot guarantee that the software has not been altered with malicious intentions.
 
 ## Known providers
+*If you host or know someone who hosts PA publicly, feel free to let me know!*  
 
-At the time of writing only [dndbox](https://planarally.dndbox.com/) is known to me to offer PlanarAlly as a public service.  If you host or know someone who hosts PA publicly, feel free to let me know!
\ No newline at end of file
+[DnDbox.com](https://dndbox.com/) v0.19.3 is hosted in Switzerland/Europe and hosts a number of projects including Planar Ally support is offerred on [reddit](reddit.com/r/dndbox/)  
```

Comment:
```suggestion
[DnDbox.com](https://dndbox.com/) v0.19.3 is hosted in Switzerland/Europe and hosts a number of projects including Planar Ally. Support is offerred on [reddit](reddit.com/r/dndbox/)  
```

Reply:
It's a lengthy of guide so in terms of readability it might be beter suited for it's own page. That said it is a useful guide for someone that wants to roll up a server in a few quick steps.  
That said it is worthwhile to mention that a docker container actually exists. Since the current docs make no mention of it.

Reply:
True, mention of the docker container would indeed be very helpful here.

Reply:
Tell you what Im gonna work on finishing the page Im on now, do some file structure changes and update the pull request give me a few minutes. Call [this one](https://hub.docker.com/r/kruptein/planarally) the official container since it's got your name on it?

Reply:
Yes that's the official image indeed, it gets automatically updated on release as well.